.github/workflows/cd.yml

```diff
@@ -0,0 +1,28 @@
+name: CD
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish_stage:
+    environment: github-pages
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - name: Set up node env and cache node modules
+        uses: actions/setup-node@v2
+          with:
+            node-version: '14'
+      - name: Install dependencies
+        run: npm i
+      - name: Build
+        run: npm run build
+      - name: Check out Github Pages branch
+      - run: |
+        git pull origin main
```

Comment:
Do we need to pull `main` before the build? Otherwise we're building the old version.

Reply:
correcto